lib/mods/import-constructors.js

```diff
@@ -4,26 +4,24 @@ const nodeTypes = require('ts-morph').TypeGuards;
 const {updateImports} = require('../imports.js');
 const {
   isGlobal,
-  getOriginal,
   getModuleNameForNode,
 } = require('../originals-helper.js');
 
 /**
- * Replaces calls to global constructors with their originals
+ * Imports originals of global constructors
  *
  * @param {Node} node - a ts-morph AST Node
  * @param {Object} imports - the current list of imports
  */
-function replaceGlobalConstructors(node, imports) {
+function importGlobalConstructors(node, imports) {
   if (nodeTypes.isNewExpression(node) &&
       isGlobal(node)) {
-    const identifier = node.getExpression();
     const moduleName = getModuleNameForNode(node);
-    const original = getOriginal(moduleName, 'constructor');
+    // original has the same name as module for constructors
+    const original = moduleName;
```

Comment:
Now there is an unused case in getOriginal that checks for type = constructor. Either we should continue to use getOriginal, or we should remove it from getOriginal

Reply:
It's not actually unused, it gets used in `generateImportText` - I compromised by leaving `constructor` in `getOriginal` so I could call it at the beginning of that function since it gets used in all the other cases.

Something I could do instead is remove it from `getOriginal` and only call that function in `generateImportText` after checking if the type is `constructor`

Reply:
Right, that seems more correct to me.

Reply:
Cool, I'll switch to that